.github/workflows/check-release.yml

```diff
@@ -1,27 +1,34 @@
 name: Check Release
+
 on:
   push:
     branches: ["*"]
   pull_request:
     branches: ["*"]
-  release:
-    types: [published]
   schedule:
     - cron: "0 0 * * *"
 
 jobs:
   check_release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup environment
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
         with:
           python_version: "3.11.x"
+
       - name: Check Release
         uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v2
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           version_spec: minor
+        env:
+          DEBUG: true  #The additional debugging output produced in this GitHub Actions workflow includes:The version of Node.js ,NPM ,jlpm ,Yarn ,Python installed on the runner.
+
+
```

Comment:
This doesn't answer exactly what `DEBUG=true` does, nor does it provide a link to any documentation proving that setting this does something.

If we don't know what `DEBUG=true` does, can you delete it from this PR?